nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/ListHDFS.java

```diff
@@ -176,7 +176,7 @@ private HDFSListing deserialize(final String serializedState) throws JsonParseEx
 
         // Build a sorted map to determine the latest possible entries
         for (final FileStatus status : statuses) {
-            if (status.getPath().getName().endsWith("_COPYING_")) {
+            if (status.getPath().getName().endsWith("_COPYING_") || status.getPath().getName().startsWith(".")) {
```

Comment:
@bbende Yes you're right! Otherwise there is the following in GetHDFS:
````java
public static final PropertyDescriptor IGNORE_DOTTED_FILES = new PropertyDescriptor.Builder()
    .name("Ignore Dotted Files")
    .description("If true, files whose names begin with a dot (\".\") will be ignored")
    .required(true)
    .allowableValues("true", "false")
    .defaultValue("true")
    .build();
````
But the filter property is much better. I'll update the PR.